crates/bazed-core/src/buffer.rs

```diff
@@ -267,9 +293,16 @@ impl Position {
 
 #[cfg(test)]
 mod test {
+    use std::sync::Once;
+
     use super::*;
+    use crate::view::Viewport;
+
+    static INIT: Once = Once::new();
+
     #[test]
     fn test_insert() {
+        INIT.call_once(|| color_eyre::install().unwrap());
```

Comment:
I feel like these should be in some init function, rather than repeating the install() for each test.

Reply:
Good idea, will do!